src/docs/arch-compact.dox

```diff
@@ -18,15 +18,22 @@ the blocks at the end of the file to the reusable blocks at the start of the fil
 The compaction process starts with a system-wide checkpoint. There are potentially many dirty blocks
 in the cache, and the intention here is to write them out and then discard previous checkpoints so
 that there are as many blocks as possible on the file's "available for reuse" list when the compaction
-starts.  The compaction logic then starts examining the files that need to be compacted. Compaction
-ignores files smaller than 1MB and computes the available bytes in the initial 80% and 90% of the file.
-Compaction doesn't proceed if at least 1MB worth of space can not be recovered. If at least 20% of
-the space is available in the first 80% of the file, the compaction process will try to rewrite
-the last 20% of the file. Else if at least 10% of the total file is available in the first 90% of
-the file, compaction is tried on the last 10% of the file. Else the file is skipped.
+starts. The compaction logic then starts examining the files that need to be compacted. Compaction
+does not proceed if one of the following conditions is met:
```

Comment:
Could we say: "...does not proceed if any of the following conditions are met:"

Reply:
Thank you, see 0b119b18000615d5915fd895b70ca723217c6b41.